### ISSUE TITLE
[Request] AST: Options to select which of Play I, II, III, and/or Minor Arcana are replaced by Astral/Umbral Draw

### ISSUE THREAD

**Issue by Starlight-alt** [MKhayle/XIVComboExpanded#422]
It would be great to be able to pick-and-choose which of the three Plays and Minor Arcana becomes Astral/Umbral Draw instead of all four being replaced when a card of their type is absent. Ideally, this would be a checklist of the four options so that any desired combination of them could be chosen.

**Comment by kaedys** [comment]
I was noodling on that while leveling Astro, but that was before the UI revamp, when it would have been more annoying to properly section the options.  My concept would probably be a feature structure a bit like:

* Replace Play with Draw when available
  * Replace Play 1 (Balance/Spear)
  * Replace Play 2 (Arrow/Bole)
  * Replace Play 3 (Spire/Ewer)
  * Replace Minor Arcana (Lord/Lady)
  * Require each Play to itself have been expended (in addition to the below settings) before it will be replaced.
  * Require Play 1 (Balance/Spear) to be expended before ANY Play actions are replaced.
  * Require Play 2 (Arrow/Bole) to be expended before ANY Play actions are replaced.
  * Require Play 3 (Spire/Ewer) to be expended before ANY Play actions are replaced.
  * Require Lord of Crowns to be expended before ANY Play actions are replaced.
  * Require Lady of Crowns to be expended before ANY Play actions are replaced.
  * Replace all Play actions with any remaining unexpended requirements if Draw is off cooldown (ie. if you set Lord as required, all Play actions set to replace would be replaced by Lord if Lord hadn't been used and Draw was off CD)

That is, however, a LOT of features to implement, and still doesn't necessarily hit _all_ the useful configurations.  But that'd _probably_ be sufficient for most of the dominant use-cases.  Notably, I found I'd want to ensure I always spent Play 1 and Lord before using Draw again, but I generally don't give a crap at all if I burned Play 2 and 3.  So I'd probably set it to replace Play 1 and Minor Arcana only, and set both Play 1 and Lord as requirements for replacement.

**Comment by MKhayle** [comment]
Yeah, the AST update predates the attributes rework and v2. I'll wait before tacking on this request because it's probably going to be a lot of headaches to test it out.

**Comment by kaedys** [comment]
Since you can use Draw and all of the Plays (including Minor Arcana) out of combat now, it's actually way easier to test than it seems.
